### DISCUSSION TITLE
Impossible to use percy:cli

### DISCUSSION THREAD

**Discussion by moonbright3** [percy/cli#842] [Support] [answered]
I follow guide in this link: https://docs.percy.io/docs/python-selenium
But when I use cmd: percy exec -> it shows error: 
zsh: command not found: percy
Below is my package.json
`{
  "name": "nop_auto",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "directories": {
    "test": "tests"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@percy/cli": "^1.0.0-beta.76"
  }
}
`
Is anyone knows why?

**Comment by Robdel12** [comment, marked as answer]
Hey @moonbright3! You would need to prefix the command with `npx` (`npx percy exec -- [...]`) or the binary directly (`./node_modules/.bin/percy exec -- [...]`). This is because the CLI is installed locally to the project, not globally (so it's not in your `$PATH`).
